autogpts/autogpt/autogpt/agents/agent.py

```diff
@@ -227,39 +261,20 @@ async def execute(
         if command_name == "human_feedback":
             result = ActionInterruptedByHuman(feedback=user_input)
             self.log_cycle_handler.log_cycle(
-                self.ai_profile.ai_name,
+                self.state.ai_profile.ai_name,
                 self.created_at,
                 self.config.cycle_count,
                 user_input,
                 USER_INPUT_FILE_NAME,
             )
 
         else:
-            for plugin in self.config.plugins:
-                if not plugin.can_handle_pre_command():
-                    continue
-                command_name, command_args = plugin.pre_command(
-                    command_name, command_args
-                )
```

Comment:
We're about to remove (most of) the old plugin interface and introduce a new one in the form of this component based agent architecture. What to do with all currently existing plugins? cc @ntindle @Torantulino

Reply:
convert the ones that are cool, kill the others

Reply:
Agreed. @kcze has also mentioned to me that he would be more than happy to help convent any useful plugins.

Reply:
Alright, so TODO in follow-up PRs:
- [ ] Remove the rest of the old plugin system
- [ ] Add loading mechanism for new plugins

And non-PR to-do's:
- [ ] Select plugins that we want to port
- [ ] Decide where to maintain plugins in the future
- [ ] Port selected plugins to new system